.circleci/config.yml

```diff
@@ -9,14 +9,27 @@ orbs:
 
 jobs:
   run-specs-with-postgres:
-    executor: solidusio_extensions/postgres
+    executor: postgres_ruby_3
```

Comment:
I think we probably want to keep this using the `solidusio_extensions/postgres` executor.